contracts/gsn/token_paymaster/TokenPaymaster.sol

```diff
@@ -6,21 +6,23 @@ import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
 import "../BasePaymaster.sol";
 import "./IUniswapV2Router02.sol";
 import "../../upgradeable_contracts/GSNForeignERC20Bridge.sol";
+import "../../upgradeable_contracts/Claimable.sol";
 
-contract TokenPaymaster is BasePaymaster {
-    address private token;
+contract TokenPaymaster is BasePaymaster, Claimable {
+    ERC20 private token;
     IUniswapV2Router02 private router;
     address private bridge;
 
     address[] private tokenWethPair = new address[](2);
     uint256 public postGasUsage = 300000;
+    uint256 public calldataSizeLimit = 1348;
```

Comment:
Why exactly this constant? If there is a way to calculate it, we should put a comment on that.

Reply:
Changed to a default GSN value - 10500. It should fit enough signatures for target bridge function. But in case it is not enough there is a setter for this value